modules/spring-security/pom.xml

```diff
@@ -39,6 +39,13 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>org.mule.services</groupId>
```

Comment:
This should be resolved as part of MULE-10837. What we discussed with @gsfernandes was to provide a test runtime that includes all required services.